src/dodal/devices/eiger.py

```diff
@@ -306,16 +316,20 @@ def _finish_arm(self) -> Status:
 
     def forward_bit_depth_to_filewriter(self):
         bit_depth = self.bit_depth.get()
-        self.odin.file_writer.data_type.put(f"UInt{bit_depth}")
+        self.odin.file_writer.data_type.set(f"UInt{bit_depth}").wait(
```

Comment:
Was this changed in attempt to fix the _wait_for_odin_status error?

Reply:
Yes, but I believe from a discussion with Dom on the beamline this is actually the "more correct" option anyway as it waits for a completion callback.

It wasn't sufficient to fix the issue on VMXm but I still think this is a good change to make.

Reply:
Yes, I agree this is more correct